worker_node/job/send.js

```diff
@@ -129,7 +143,8 @@ class Send extends Base {
   }
 
   async sendTxs() {
-    const delayTime = this.config.duration / this.config.numberOfTransactions * 1000;
+    const delayAdjustmentRate = 0.97;
+    const delayTime = (this.config.duration / this.config.numberOfTransactions * 1000) * delayAdjustmentRate;
```

Comment:
As discussed offline, let's try removing this `delayAdjustmentRate` and instead measure the more exact transmission times in the promises! Thanks for the discussion. If it's too much hassle, we can skip for now though.

Reply:
Done! I added `targetTestEndTime` and use it in delay!